Master-Game/Entities/PlayerEntity.cs

```diff
@@ -5,14 +5,14 @@ namespace MasterGame.Entities
 {
     public class PlayerEntity : BaseEntity
     {
-        public PlayerEntity(string name, float healthPoints)
-           : base(EntityType.Player, name, healthPoints, true, true, true, true)
+        public PlayerEntity(string name, float healthPoints, float maxHealthPoints)
+           : base(EntityType.Player, name, healthPoints, maxHealthPoints, true, true, true, true)
```

Comment:
instead of putting maxHP and hp just use one especailly since in reset you are setting HP = MAX.

Reply:
yup...we have a third property now defaultHP